appveyor.yml

```diff
@@ -10,28 +15,36 @@ environment:
   CHROMAPRINT_FPCALC_VERSION: 1.4.2
   DISCID_VERSION: 0.6.1
   matrix:
-  - PYTHON: C:\Python36
-    PYTHON_VERSION: 3.6.1
-    PYTHON_ARCH: 32
-  - PYTHON: C:\Python35
-    PYTHON_VERSION: 3.5.3
-    PYTHON_ARCH: 32
-build: off
-test_script:
+  - PYTHON: C:\Python35-x64
+    python_version: 3.5.3
+    python_arch: 64
+  - PYTHON: C:\Python36-x64
+    PYTHON_VERSION: 3.6.4
+    PYTHON_ARCH: 64
+build_script:
 - cmd: >-
     appveyor DownloadFile https://github.com/acoustid/chromaprint/releases/download/v%CHROMAPRINT_FPCALC_VERSION%/chromaprint-fpcalc-%CHROMAPRINT_FPCALC_VERSION%-windows-i686.zip -FileName fpcalc.zip
 
+    appveyor DownloadFile https://indy.fulgan.com/SSL/openssl-%OPENSSL_VERSION%-x64_86-win64.zip -FileName openssl.zip
```

Comment:
Where is OPENSSL_VERSION defined? I can't find it on https://www.appveyor.com/docs/environment-variables/ and I think it's also not visible in the build log (https://ci.appveyor.com/project/MetaBrainz/picard/build/2.0.0.dev.2.49/job/8w22578qj5bf7wgh).

Reply:
Its configurable in AppVeyor settings along with the other env variables.

Reply:
Could we move this to the environment section in this file or echo the version at some point? Otherwise it's nearly impossible to figure out which openssl version is downloaded for normal people. At the moment it appears to be set to 1.0.2, but 1.0.2n is already available.